### ISSUE TITLE
How to predict the single sample after training the model

### ISSUE THREAD

**Issue by lasso007** [hritik25/Dynamic-CNN-for-Modelling-Sentences#3]
In "architecture.py":

> layerInput = lasagne.layers.InputLayer(shape = (batchSize, None))

The input of the model is limited with batchSize, but how can I feed one sample into this model?

**Comment by hritik25** [comment]
Hi,
To feed one sample, you give it a batch size of 1. That is how I trained it, so as to avoid padding. Hope that helps.

**Comment by lasso007** [comment]
Oh, I get it, thank you!  
However, what is the accuracy of the model trained with batch_size=1 ?  and the results stated in the paper "A Convolutional Neural Network for Modelling Sentences" is very good.

**Comment by hritik25** [comment]
When I trained this model, I did not have a powerful GPU. With 1 epoch of training, I got an accuracy of 78% on the binary sentiment classification task, the results in the paper reported an accuracy of 87% with 30 epochs of training.
